backend/src/main/java/de/echochamber/backend/controller/PollController.java

```diff
@@ -66,98 +66,51 @@ public PollController(PollService pollsService, UserService userService, AnswerS
                     .chosenAnswer(possibleAnswerEntity)
                     .user(userEntityOptional.get())
                     .build();
-            return ok(map(pollsService.giveAnswer(givenAnswer)));
+            return ok(mapper.map(pollsService.giveAnswer(givenAnswer)));
         }
         throw new EntityNotFoundException("User not found");
     }
 
-    @GetMapping
+    @GetMapping(value ="/all")
     public ResponseEntity<List<Poll>> findAll() {
         List<PollEntity> pollEntities = pollsService.findAll();
-        return ok(map(pollEntities));
+        return ok(mapper.map(pollEntities));
     }
 
-    private List<Poll> map(List<PollEntity> pollEntities) {
-        List<Poll> polls = new ArrayList<>();
-        for (PollEntity pollEntity : pollEntities) {
-            polls.add(map(pollEntity));
+    @GetMapping(value = "/open")
```

Comment:
This could be a query param of get all, so that you have an url like `/polls?filter=open`?